js/simple/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
   },
   "devDependencies": {
-    "azure-iot-gateway": "~1.1.4"
+    "azure-iot-gateway": "~1.1.4",
+    "azure-iot-device-amqp": "~1.1.17"
```

Comment:
This is where this iothub_writer coming from:
https://github.com/Azure/iot-edge/blob/master/samples/nodejs_simple_sample/nodejs_modules/iothub_writer.js

Reply:
I have a sample working with MQTT if we want to switch.

Reply:
@JSeb225 do you think that we should tell customer to use MQTT or AMQP ?

Reply:
AMQP is important for the connection from the gateway to IoT Hub because it can multiplex telemetry from multiple devices onto a single TCP connection. The IoT Hub module in our gateway SDK can be configured to do MQTT instead, but then you don’t get multiplexing—the module will open a new connection for every device.